packages/components/e2e/component-objects/src/main/java/org/talend/component/list/table/Item.java

```diff
@@ -79,7 +84,11 @@ public Cell getCell(final String columnKey) {
      * Click on item title.
      */
     public void clickOnTitle() {
-        getTitle().click();
+        final WebElement title = getTitle();
+        final Actions actions = new Actions(driver)
+                .moveToElement(title)
+                .click(title);
```

Comment:
We need to precise that we want to click on the title element. A simple `.click()` is caught by the Table grid div, event if we to mouse to the button.
That's because of react-virtualized that set a tabindex on the grid div.